project/recursive_sorting.py

```diff
@@ -60,13 +60,14 @@ def median_of_3(arr, low, high):
     
     def partition(arr, low, high):
```

Comment:
Is this running fine on your computer, McKay? I tried running your code myself but it doesn't work (I may have missed something)